administrator/components/com_joomlaupdate/Model/UpdateModel.php

```diff
@@ -830,11 +836,19 @@ public function finaliseUpgrade()
 	 */
 	public function cleanUp()
```

Comment:
Maybe we can also expand the comment for this method. For example: 
`Removes the extracted package file and creates information regarding template overrides.`